ground_truth_labeling_jobs/bring_your_own_model_for_sagemaker_labeling_workflows_with_active_learning/bring_your_own_model_for_sagemaker_labeling_workflows_with_active_learning.ipynb

```diff
@@ -0,0 +1,715 @@
+{
```

Comment:
<p>Should we add a warning in bold to check if the push succeeded? I'm worried that they might miss the step where they need to add the ECR permissions to the notebook instance role, and they will only see it here. Also, the failure in that case is nearly silent, it just says something like "access denied" at the end of the cell output.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1121/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>